alerts/openshift-container-storage-operator/StorageAutoScalingCapacityReached.md

```diff
@@ -0,0 +1,30 @@
+# StorageAutoScalingCapacityReached
+
+## Meaning
+
+The Storage Capacity Limit is the maximum storage that can be provisioned for
+the system. `StorageAutoScalingCapacityReached` alert is fired when the complete
+storage space has been utilized and hence additional storage cannot be allocated
+for the device class in the particular storage cluster.
+
+## Impact
+
+The storage cannot be auto-scaled at the moment.
+
+## Diagnosis
+
+Retrieve status of `storageCapacityLimitReached` from the StorageAutoScaler CR.
+
```

Comment:
resource name and resource namespace would be provided by alert, maybe we want to specify or not specify it